live-build/misc/live-build-hooks/82-upgrade-repository.binary

```diff
@@ -149,6 +149,22 @@ aptly repo add upgrade-repository ./*.deb
 
 aptly publish repo -skip-signing upgrade-repository
 
-tar -czf "$APPLIANCE_VARIANT.upgrade.tar.gz" -C upgrade-scripts . -C ~/.aptly .
+#
+# We also want to include a file about the current image's version metadata:
+#   DLPX_VERSION: The application (product) version.
+#   DLPX_MIN_VERSION: The minimum version an engine must be on, in order to
+#     upgrade to this image.
+#   DLPX_MIN_REBOOTED_VERSION: The minimum version an engine did a reboot
+#     upgrade to, in order to upgrade to this image.
+#
+rm -rf version.info
```

Comment:
Why do we delete this beforehand? The subsequent `cat` operation already uses `>` to replace `version.info` if it already exists.